hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1602,15 +1602,15 @@ func (r *HostedClusterReconciler) reconcileCAPIManager(ctx context.Context, crea
 
 	// Reconcile CAPI manager deployment
 	capiImage, err := hyperutil.GetPayloadImage(ctx, hcluster, ImageStreamCAPI, pullSecretBytes)
```

Comment:
Should we run this only if capi image is not set in env or annotations?

Reply:
Yeah that makes sense to me. I had originally left it here to keep the order of how it was set. I’ll make the change and get it pushed up.